emcc.py

```diff
@@ -1037,7 +1037,7 @@ def run(args):
     process_libraries(state.libs, state.lib_dirs, [])
     if len(input_files) != 1:
       exit_with_error('--post-link requires a single input file')
-    phase_post_link(options, input_files[0][1], wasm_target, target)
+    phase_post_link(options, input_files[0][1], wasm_target, target, state)
```

Comment:
Can you make state the second argument after options to match other phases that take both?